src/App.tsx

```diff
@@ -2,45 +1,87 @@
+//import classNames from 'classnames';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
 
-import { PostsList } from './components/PostsList';
-import { PostDetails } from './components/PostDetails';
+//import { PostDetails } from './components/PostDetails';
```

Comment:
The `PostDetails` component is commented out. If this component is intended to be part of the application, consider uncommenting it and ensuring it is used correctly.